docs.old/extensions/manifest.md

```diff
@@ -16,7 +18,7 @@ In this configuration file, you have to define the following properties :
 | **description** | Translatable string (required) | The description of the extension in multiple language                                    |
 | **icon**        | Translatable string (required) | The public link to the icon of the extension in multiple language                        |
 | **links**       | JSON Object                    | The useful HTTP links related to the extension.                                          |
-| **components**  | Array of components (required) | yumm                                                                                     |
+| **components**  | Array of components (required) | yumm                                                                                    |
```

Comment:
yumm?

Reply:
this one will not be used anymore. I kept it to retrieve the previous documentation